apps/api/env/schema.mjs

```diff
@@ -23,6 +25,7 @@ export const serverSchema = z.object({
 export const clientSchema = z.object({
   // NEXT_PUBLIC_CLIENTVAR: z.string(),
   NEXT_PUBLIC_GOOGLE_ANALYTICS_MID: z.string().optional(),
+  NEXT_PUBLIC_SUPER_TOKENS_APP_NAME: z.string().optional(),
```

Comment:
this needs to be added to clientEnv too, otherwise it will not be included in the compiled client